packages/devtools-network-console/src/host/vscode-protocol-host.ts

```diff
@@ -270,7 +270,7 @@ export default class VsCodeProtocolHost implements INetConsoleHost {
         const deserialized = deserializeFromHost(message.requestId, message.request);
         globalDispatch(loadRequestAction(message.requestId, deserialized, message.requiresSaveAs));
 
-        // Environment set via `LOAD_REQUEST` deprecated in v0.11.0-preview
+        // Environment set via `LOAD_REQUEST` deprecated in v0.11.1-preview
```

Comment:
This change should be reverted. Also consider removing this commented-out code altogether.